scripts/gdal_helper.py

```diff
@@ -67,8 +71,8 @@ def run_gdal(command: List[str], input_file: str = "", output_file: str = "") ->
             capture_output=True,
         )
     except subprocess.CalledProcessError as cpe:
-        get_log().error("run_gdal_failed", command=command_to_string(command))
-        raise cpe
+        get_log().error("run_gdal_failed", command=command_to_string(command), error=str(cpe.stderr, "utf-8"))
+        raise GDALExecutionException(f"GDAL {str(cpe.stderr, 'utf-8')}") from cpe
     get_log().debug("run_gdal_translate_succeded", command=command_to_string(command))
```

Comment:
should check stderr for any warnings/issues too.

also stdout might be interesting to dump as a log message?

Reply:
> should check stderr for any warnings/issues too.
Yes, it is actually checked in the script itself but it should be done here you're right.

> also stdout might be interesting to dump as a log message?
:+1: